porting/index.md

```diff
@@ -65,13 +72,66 @@ You can also specify `-j[num]` as it will fetch the files simultaneously. This d
 
 Parts that are needed:
-- Device tree
-- Kernel source
+- Device tree (e.g. from [LineageOS](https://github.com/LineageOS))
+- Kernel source (e.g. also from LineageOS)
 - Vendor tree (e.g. from [TheMuppets](https://github.com/TheMuppets))
 
+LineageOS and TheMuppets have a lot of devices already prepared, but it is not mandatory to use only their sources. Basically any tree capable of producing a working (Android) ROM should suffice.
+
 You might have to check `lineage.dependencies`/`cm.dependencies` that is included in every LineageOS/CyanogenMod device repo for additional repositories.
 
-**//TODO: Include documentation about local_manifests.**
+### Create a local manifest ###
+
+The repo tool will accept additional repositories to be synced on top of the ones defined by Halium already. Since you are porting for a new device, you also need to do this work. After you have a working configuration your local manifest.xml can go into an issue in the [Halium/projectmanagement repo](https://github.com/Halium/projectmanagement/issues?q=is%3Aissue%20%5Bdevice-port%5D%20in%3Atitle) .
+
+Create additional entries for the repo tool to download the required parts automatically:
+
+```
+cd <BUILDDIR>/.repo && mkdir local_manifests && cd local_manifests
+```
+
+Create a new file called `<VENDOR>_<CODENAME>.xml` and open it in your favorite editor.
+
+For the time being, add entries to the created xml file as follows:
+
+```
+<?xml version="1.0" encoding="UTF-8"?>
+<manifest>
+<!-- Example for a device with one specific vendor repo, and 2 repos for the device, a common and a specific one. At the end, a common kernel repo -->
+  <project path="vendor/samsung" name="proprietary_vendor_samsung" remote="them" />
+  <project name="android_device_samsung_smdk4412-common" path="device/samsung/smdk4412-common" remote="los" />
+  <project name="android_device_samsung_i9300" path="device/samsung/i9300" remote="los" />
+  <project name="android_kernel_samsung_smdk4412" path="kernel/samsung/smdk4412" remote="los" />
+  
+  <!-- Example for additional repos gathered from the dependencies files in the device or vendor repo -->
+  <project name="android_hardware_samsung" path="hardware/samsung" remote="los" />
+  <project name="android_external_stlport" path="external/stlport" remote="los" />
+
+</manifest>
+```
+
+The remote properties "los" and "them" are shortcuts for the already defined remotes that Halium provides. They can be reviewed by looking into default.xml in the .repo/manifests directory (It is also symlinked to .repo/manifest.xml). Depending on which Halium branch you used, default.xml will be either set up for [LineageOS 14.1](https://github.com/Halium/android/blob/halium-7.1/default.xml) or [CyanogenMod 12.1 ](https://github.com/Halium/android/blob/halium-5.1/default.xml). You can create references to new remotes by adding the following snippet in front of any project definition:
```

Comment:
There's an extra whitespace after `CyanogenMod 12.1`, probably doesn't change the rendered output but still